css/css-text/text-align/reference/text-align-end-ref-016.html

```diff
@@ -1,20 +1,21 @@
 <!DOCTYPE html>
-<html  lang="en" >
+<html lang="en">
 <head>
 <meta charset="utf-8">
 <title>text-align: end, pre, dir=auto</title>
 <link rel='author' title='Richard Ishida' href='mailto:ishida@w3.org'>
-<link rel="stylesheet" type="text/css" href="/fonts/ahem.css" />
+<link rel="stylesheet" type="text/css" href="/fonts/ahem.css">
 <style type='text/css'>
-.test, .ref { border: 1px solid orange;  margin: 20px; width: 290px; color: orange; font: 24px/24px Ahem; }
-.ref { position: relative;  height: 48px; }
-.rb1 { position: absolute; top: 0; right: 0; background-color: orange; width: 72px;  height: 24px; }
-.rb2 { position: absolute; bottom: 0; left: 0; background-color: orange; width: 72px;  height: 24px; }
+.ref1, .ref2 { border-left: 1px solid orange; border-right: 1px solid orange; margin-left: 20px; margin-right: 20px; width: 300px; color: orange; font: 25px/1 Ahem; }
```

Comment:
border-inline seems to work with all major browsers.  It could be useful here to make the code more readable.

Reply:
Done.